downgrade ember-tether

we're on version 0.4.0 right now, downgrade to version 0.3.1 b/c of unmet dependencies with ember-tooltips.

```bash
stefan@pittiplatsch: ~/dev/projects/frontend on master
$ npm install
npm WARN ember-tooltips@2.8.0 requires a peer of ember-tether@0.3.1 but none was installed.

stefan@pittiplatsch: ~/dev/projects/frontend on master
$ nvm --version; npm --version; node --version;
0.31.6
3.10.3
v6.4.0

stefan@pittiplatsch: ~/dev/projects/frontend on master
$ npm list ember-tooltips
ilios@1.30.2 /home/stefan/dev/projects/frontend
├── UNMET PEER DEPENDENCY ember-tether@0.4.0
└── ember-tooltips@2.8.0 

npm ERR! peer dep missing: ember-tether@0.3.1, required by ember-tooltips@2.8.0
```

--- a/package.json
+++ b/package.json
@@ -109,7 +109,9 @@
   },
   "greenkeeper": {
     "ignore": [
-      "ember-cli-mirage"
+      "ember-cli-mirage",
+      "ember-tether"
+
     ]
   },
   "private": true

--- a/package.json
+++ b/package.json
@@ -82,7 +82,7 @@
     "ember-rl-dropdown": "0.10.0",
     "ember-simple-auth": "1.1.0",
     "ember-simple-auth-token": "2.1.0",
-    "ember-tether": "0.4.0",
+    "ember-tether": "0.3.1",
     "ember-tooltips": "2.8.0",
     "ember-truth-helpers": "1.3.0",
     "ember-uploader": "1.2.3",

